code/index.html

```diff
@@ -1,21 +1,44 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>somewhere between paper an skin</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
 
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
-  </head>
-  <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+  <nav>
+    <div class="nav"></div>
+    <ul>
+      <li><a href="#">/art work</a></li>
+      <li><a href="#">/tattoo art</a></li>
+      <li><a href="#">/about</a></li>
+      <li><a href="https://instagram.com/jessy.vagabunda?utm_medium=copy_link">/insta</a></li>
+      </ul>
```

Comment:
Be careful with i**ndentation** here. 🔍 The rule to follow is that if there's a new block, then you indent one more level. At Technigo, we use 2 spaces for indentation. The list's closing tag should be in line with its opening tag. 🙂